src/main/java/com/task/amazon/Main.java

```diff
@@ -0,0 +1,14 @@
+package com.task.amazon;
+
+import com.task.amazon.utils.ReaderService;
+import com.task.amazon.utils.impl.FileReaderService;
+
+import java.util.List;
+
+public class Main {
```

Comment:
@karsv isn't it a SpringBoot app? We should have only one main method. If this is a test case - please move it to a corresponding folder